charts/umbrella/README.md

```diff
@@ -123,6 +123,10 @@ If you still face DNS issues, add the hosts to your /etc/hosts file:
 192.168.49.2    dataprovider-dataplane.tx.test
 192.168.49.2    dataconsumer-2-dataplane.tx.test
 192.168.49.2    dataconsumer-2-controlplane.tx.test
+192.168.49.2    partners-pool.tx.test
+192.168.49.2    partners-gate.tx.test
+192.168.49.2    partners-orchestrator.tx.test
+192.168.49.2    partners-cleaning.tx.test
```

Comment:
The cleaning service does not have a API and does not need to be exposed. Have a look here to see the recommended settings: #108